src/cryptonote_config.h

```diff
@@ -145,6 +145,7 @@
 #define P2P_DEFAULT_PEERS_IN_HANDSHAKE                  250
 #define P2P_MAX_PEERS_IN_HANDSHAKE                      250
 #define P2P_DEFAULT_CONNECTION_TIMEOUT                  5000       //5 seconds
+#define P2P_DEFAULT_REQUEST_TIMEOUT                     (P2P_DEFAULT_CONNECTION_TIMEOUT/1000) // 5 seconds
```

Comment:
5s seems like not a lot of time, potentially a cause of uptick in bans when running tx relay v2..

E.g. let's say you have 5 128-input txs to receive.. even if you assume 0 latency to receive the tx after requesting it, it could take at least 15s to process the last tx in that batch and mark it received